### PR TITLE
[Tech] business value redesign

### PR DESCRIPTION
**Pull request by bhlynsky**
### Here is provided design variants (updated)

**1. "Stains" with sharp transition** (main option)

- Desktop 
image
- Mobile 
image

**2. Big stain with smooth block transition**  (proposal)

- Desktop 
 image
- Mobile 
image

**3. Simple horizontal gradient** (analog for previous ones)

- Desktop
 image
- Mobile 
image

[_old designs for reference_](https://user-images.githubusercontent.com/86774452/174602348-bdfce31f-0e79-48a2-91ab-4d89692733eb.png)

### CODE REVIEW

File: docs/website/src/landing/business-value/business-value.less [hunk 1]
```diff
@@ -18,6 +15,47 @@
       padding: 0.5rem;
       font-size: 14px;
     }
+
+    &.design-1 {
+      background-color: @landing-blue;
+      color: #fff;
```
Comment (NastaLeo) [inline docs/website/src/landing/business-value/business-value.less:21]: If i'm not mistaken, there are variables:
@landing-bg-color: #fff;
@landing-fg-color: #000;

**Review by Bogdan9817 — APPROVED** [review]

**Comment by Bogdan9817** [comment]
imho, 3rd and 8th

**Comment by abarmina** [comment]
I liked option 3 better (by colors and readability)
as an idea - can we try something with a vertical gradient with previous and next block (or just one side) or some stains?
*as I remember, we didn't want to use purple as the main color

just pictures with stains. yes, this is a difficult task, since none of us know how it should look like =)
image
image

**Review by ala-n — APPROVED** [review]
Approved for the first version of implementation

**Comment by ala-n** [comment]
Merged but needs to be updated separately (remove demo samples)

**Pull request merged**